modules/tinymce/src/core/main/ts/api/dom/TableResizeHandler.ts

```diff
@@ -6,48 +6,51 @@
  */
 
 import { Arr, Optional, Strings } from '@ephox/katamari';
-import { Adjustments, ResizeBehaviour, ResizeWire, Sizes, TableGridSize, TableResize } from '@ephox/snooker';
+import { Adjustments, ResizeBehaviour, ResizeWire, Sizes, TableConversions, TableGridSize, TableLookup, TableResize, Warehouse } from '@ephox/snooker';
 import { Attribute, Css, SugarElement } from '@ephox/sugar';
 
-import Editor from 'tinymce/core/api/Editor';
-
-import * as Events from '../api/Events';
-import * as Options from '../api/Options';
-import * as Util from '../core/Util';
-import * as TableSize from '../queries/TableSize';
-import { enforcePercentage, enforcePixels, syncPixels } from './EnforceUnit';
-import * as TableWire from './TableWire';
-
-export interface ResizeHandler {
-  readonly lazyResize: () => Optional<TableResize>;
-  readonly lazyWire: () => ResizeWire;
+import * as NodeType from '../../dom/NodeType';
+import * as TableSize from '../../table/TableSize';
+import * as Util from '../../table/TableUtil';
+import * as TableWire from '../../table/TableWire';
+import Editor from '../Editor';
+import * as Options from '../Options';
+import * as Events from '../TableEvents';
+
+export interface TableResizeHandler {
+  readonly refreshBars: (table: HTMLTableElement) => void;
+  readonly hideBars: () => void;
+  readonly showBars: () => void;
```

Comment:
I don't really like the name bars here on a public API as it's not overly clear... can these be called `handles` or something else instead?

Reply:
That makes sense. Have renamed to have a postfix of `Handles` instead of `Bars`